client/src/app/site/pages/meetings/modules/projector/modules/slides/components/poll-slide/components/poll-slide.component.scss

```diff
@@ -120,27 +121,37 @@
             }
         }
 
-        :not(:last-child):not(h3) {
+        :not(:last-child):not(.structure-level) {
             border-bottom: 1px solid rgba(128, 128, 128, 0.2);
         }
 
-        .vote-entry:has(+ h3) {
+        :last-child:not(.structure-level) {
+            margin-bottom: 1px;
+        }
+
+        .vote-entry:has(+ .structure-level) {
             border-bottom: 0;
         }
 
-        h3.structure-level {
-            padding-top: 10px;
-            padding-bottom: 1px;
-            margin-top: 0;
-            margin-bottom: 0 !important;
-            line-height: 1;
-            font-size: 18px;
-            font-weight: 500;
-
-            // setting for the ellipsis for long structure levels
-            white-space: nowrap;
-            overflow: hidden;
-            text-overflow: ellipsis;
+        .structure-level {
+            margin-bottom: 2px !important;
+            min-height: 29px;
```

Comment:
The height must be fixed for the column size calculation to work.

Reply:
Used height, Idk what you mean with text-overflow, I am using it in the h3 style and it works